server/monitor/src/main/java/org/apache/accumulo/monitor/util/logging/AccumuloMonitorAppender.java

```diff
@@ -122,8 +122,9 @@ public void append(final LogEvent event) {
 
         var req = HttpRequest.newBuilder(uri).POST(BodyPublishers.ofString(jsonEvent, UTF_8))
             .setHeader("Content-Type", "application/json").build();
-        @SuppressWarnings("unused")
+
         var future = httpClient.sendAsync(req, BodyHandlers.discarding());
+        future.get();
```

Comment:
You described this in #4877, but I didn't get it until I saw the code. I think this is assuming that if this appender blocks appends, then that will put back pressure on the log4j async / lmax disruptor stuff, and we won't have as many client threads still doing work sending log messages to the monitor all at once from a single process.

This seems like correct reasoning, but I'm not 100% sure.

Implementation-wise, the one-liner version of this would be:

```suggestion
        httpClient.sendAsync(req, BodyHandlers.discarding()).get();
```

Or maybe there's a more efficient method to call on the httpClient. I'm not sure.

Reply:
> You described this in https://github.com/apache/accumulo/issues/4877, but I didn't get it until I saw the code.

That is why I made the PR.  Figured the code was much better than english for this.  

> I think this is assuming that if this appender blocks appends, then that will put back pressure on the log4j async / lmax disruptor stuff

That is my hope, but not sure of the overall implications of this change.

Reply:
The only thing I can think to do with this change to get a better sense of how it will actually behave is to do some testing.

Reply:
Also, we could probably just call the send method on the http client instead of the sendAsyc method.  Either way, need to see what the overall change in behavior is.